# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/tests/common.js
# Commit: 4bed3a0e2332
# Full Hash: 4bed3a0e2332f24ffe38f08e234c8898c42249ee
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2016-11-07 02:21:38
# Regressor Bug: 1290021
# File Overlap Count: 1
# Description:
#   Bug 1290021 - Implement a prototype version of Houdini "Worklets Level 1" spec - part 1 - WebIDL interface, r=smaug
# ==============================================================================

diff -r d84108f190ee -r 4bed3a0e2332 dom/worklet/tests/common.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/worklet/tests/common.js	Sun Nov 06 09:53:34 2016 +0100
@@ -0,0 +1,14 @@
+function loadTest(file) {
+  var iframe = document.createElement('iframe');
+  iframe.src = file;
+
+  document.body.appendChild(iframe);
+}
+
+function setupTest() {
+  window.SimpleTest = parent.SimpleTest;
+  window.is = parent.is;
+  window.isnot = parent.isnot;
+  window.ok = parent.ok;
+  window.info = parent.info;
+}